src/client/App.css

```diff
@@ -67,8 +67,7 @@ img {
   z-index: 0;
 }
 .bodywrapper {
-  position: relative;
-  padding: 5%;
+  position: absolute;
```

Comment:
You generally want to avoid using `position: absolute;` if you can avoid it. Reason being that the `absolute` property means taking the element out of the general page flow so other elements around it will not be aware of the absolute positioned elements size and position. I'm not sure what you're trying to achieve here but it seems to break other pages, so I think it's back to the drawing board :)

Agree with priyanka that you could apply the styling to a custom class, but I feel like it might be treating symptoms rather than the root cause.

Generally I see a lot of layout being re-implemented on every single page in the application, despite the layout on each page being almost the same. I would strive for making a single component that implements the layout and comes with the styling needed.